src/qrl/core/BlockHeader.py

```diff
@@ -44,6 +49,11 @@ def timestamp(self):
 
     @property
     def headerhash(self):
+        """
+        the Qryptonight hash of of BlockHeader's mining_blob representation.
```

Comment:
1. "the" First letter should be capital.
2. "of of" duplicate word
3. This comment doesnt make much sense, needs rewording.